homeassistant/components/smart_meter_texas/__init__.py

```diff
@@ -2,7 +2,7 @@
 import asyncio
 import logging
```

Comment:
```suggestion
import logging
import ssl
```

Looks like you still need it for the typing on `ssl.SSLContext`